### PR TITLE
Kevin/new statuses

### PR DESCRIPTION
**Pull request by owocki**
# What

Updates the states of the gitcoin system  to reflect the states that @vs77bb put togethter:

https://docs.google.com/document/d/1TDrTtfBwetGqL5-41u5a_4-1ttz6r0wA5qa0NZhDOuA/edit

made these changes

```
express interest => claim work => start work
fulfill work => submit work
```

and then regression tested both flows.. both worked

gonna move these ones into a v2 pr
```
checkin every 3 days IFF user has started work w. submitter
    - are you still working on this? 
checkin every 3 days IFF user has submitted work w. fulfiller
    - are you still working on this? 
1:n bounty:fulfillment
    - make it so that the UI supports multiple fulfillments
    - (maybe?) allow off-chain rejections. (v2)
```

### CODE REVIEW

File: app/dashboard/migrations/0034_auto_20180129_1344.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.0 on 2018-01-29 20:44
```
Comment (mbeacom) [inline app/dashboard/migrations/0034_auto_20180129_1344.py]: This will yield a migration conflict.  Can you delete this file and re-run makemigrations?

Comment (owocki) [reply app/dashboard/migrations/0034_auto_20180129_1344.py]: 8200bfe2eeab57ef69fdc108b34fe1fedd47c721

**Comment by jasonrhaas** [comment]
image

Wow!

**Comment by owocki** [comment]
@mbeacom mind doing a fresh review of https://github.com/gitcoinco/web/pull/319/commits/18f8815120ff1935c210ca5c44d20b8967bc3f30 ?

Here are the two emails in taht commit: 

removal_notice
warning

**Comment by owocki** [comment]
> Are there any adverse side effects with the existing data? These changes definitely affect the multilple bounties stuff I'm working on so I will want to test with this code I think.

`./manage.py refresh_bounties`, which runs every 15 minutes... will catch the existing bounties and migrate them to their new status'

fwiw... i tested a few existing bounties during my regression test..

**Comment by jasonrhaas** [comment]
Doesn't seem to be rendering the HTML in the tooltip.

image

**Comment by owocki** [comment]
@jasonrhaas hard refresh... tooltip.js has been updated

**Comment by owocki** [comment]
@jasonrhaas 1143addf5ccfc07b5f09bc906d16654f6d406a51 should fix that

**Comment by jasonrhaas** [comment]
@owocki hmm Something is still funky about how that "Work Started" status is working.  Doesn't seem to be updating properly.

File: app/retail/templates/emails/render_bounty_startwork_expired.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{% load humanize %}
```
Comment (mbeacom) [inline app/retail/templates/emails/render_bounty_startwork_expired.txt]: 🙌

File: app/marketing/management/commands/expiration_start_work.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+from dashboard.models import Interest, Bounty
+
+from github.utils import get_issue_comments, org_name, repo_name, issue_number
+
+from marketing.mails import bounty_startwork_expire_warning, bounty_startwork_expired
+
+
+class Command(BaseCommand):
+
+    help = 'lets a user know that they expressed interest in an issue and kicks them to do something about it'
```
Comment (mbeacom) [inline app/marketing/management/commands/expiration_start_work.py]: 😂

**Comment by owocki** [comment]
> Something is still funky about how that "Work Started" status is working. Doesn't seem to be updating properly.

id need to know repro steps here.. seems to be working on my local

**Pull request merged**